airbyte-integrations/base-python/airbyte_protocol/entrypoint.py

```diff
@@ -85,19 +86,17 @@ def start(self, args):
             if cmd == "check":
                 check_result = self.source.check(logger, config_container)
 
-                if check_result.successful:
+                if check_result.status == Status.SUCCEEDED:
```

Comment:
@sherifnada If i understand right, this change is not compatible with what is currently in the `template/python-source` for the ´ check' function anymore, right?

`AirbyteCheckResponse` has to be changed to `AirbyteConnectionStatus` in that file too?

Reply:
@ChristopheDuong Correct - good catch